controllers/mirrorpeer_controller.go

```diff
@@ -85,6 +88,57 @@ func (r *MirrorPeerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}
 	logger.V(2).Info("All validations for MirrorPeer passed", "MirrorPeer", req.NamespacedName)
+
+	// Define RamenDRPolicy object.
+	ramenDRPolicy := ramendrv1alpha1.DRPolicy{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "DRPolicy",
+			APIVersion: "v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "drpolicy",
+		},
+	}
+	// Make RamenDRPolicy entries reuseable.
+	ramenDRPolicyKeysAndValues := make([]interface{}, 2)
+	ramenDRPolicyKeysAndValues[0] = "RamenDR Policy"
```

Comment:
```suggestion
	ramenDRPolicyKeysAndValues[0] = "DRPolicy"
```

Reply:
I browsed through the RamenDR repository and found out that they were using an hour as the default value for this at [a number of places](https://github.com/RamenDR/ramen/search?p=1&q=SchedulingInterval). Should I remove it or change it to something else?